src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,65 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: Tab[],
+  selectedTab: Tab,
+  setSelectedTab: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTab,
```

Comment:
Point 4: 'Pass the selectedTabId as a prop to the Tabs, the specified tab should be selected if possible (otherwise the first tab is selected).'